README.md

```diff
@@ -54,3 +54,25 @@ TEMPLATES = [
     }
 ]
 ```
+
+A custom folder to load the templates can be specified. This allows you to keep the URL acessible templates separately from template includes.
+
+``` python
+# As a subfolder in templates. Only searching inside `templates/pages/`
+TEMPLATE_FINDER_DIR = 'pages'
```

Comment:
Yeah, I don't mind doing that. I was struggling to decide best name and I went with DIR because templates had DIRS.

Leaning towards PATH.